tests/fbe_factory.pm

```diff
@@ -23,8 +23,14 @@ sub run {
     assert_screen('text_console_login', 10);
     send_key("ctrl-alt-f1");
 
-    # check that the FBE welcome screen reappears
-    assert_screen('fbe_welcome', 10);
+    # check that the FBE welcome screen reappears; choose "Indonesian"
+    assert_and_click('fbe_welcome_choose_indonesian', 'left', 10);
+
+    # wait for the "Next" button to change to Indonesian and click it
+    assert_and_click('fbe_welcome_indonesian', 'left', 10);
+
+    assert_and_click('fbe_keyboard_indonesian', 'left', 10);
+    assert_screen('fbe_license_indonesian', 10);
```

Comment:
I feel a bit more strongly than with the `text_console_login` check above that this language changing stuff should be in a separate `fbe_*` test of its own. Otherwise `fbe_factory` is going to become very, very long.

Reply:
and now I have read this comment. so, you think there should be separate test cases for:

* does the factory test dialog work
* does vt-switching work
* does language switching work
etc?

Reply:
Yeah, I think so. Partially because chaining lots of little tests together into a single `.pm` file makes them all just a little bit less maintainable due to the potential accumulation of state on the SUT, partially because splitting them allows the test server to parallelise them (it currently has two workers, and this number could be increased in future), and partially because it means failures can be attributed to the right test so we can track them over time if we ever feel so inclined.